src/screens/Assets/PPNView.js

```diff
@@ -0,0 +1,359 @@
+              <ValueInFiat>
+                {`${currencySymbol} ${formattedAmountInFiat}`}
+              </ValueInFiat>
+            </BalanceWrapper>
+          </AddonWrapper>
+        }
+        rightColumnInnerStyle={{ flexDirection: 'row' }}
+      />
+    );
+  };
+
+  renderHeader = () => {
+    const {
+      assetsOnNetwork,
+      navigation,
+      rates,
+      baseFiatCurrency,
+      balances,
+    } = this.props;
+    const portfolioBalances = calculatePortfolioBalance(assetsOnNetwork, rates, balances);
+    const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
+    const PPNBalance = formatMoney(portfolioBalances[fiatCurrency] || 0);
+    const currencySymbol = getCurrencySymbol(fiatCurrency);
+
+    return (
+      <ListHeaderWrapper>
+        <HeaderTitle>{`Balance ${currencySymbol} ${PPNBalance}`}</HeaderTitle>
+        <HeaderButton
+          onPress={() => navigation.navigate(SETTLE_BALANCE)}
+          disabled={!Object.keys(assetsOnNetwork).length}
+        >
+          <ButtonText disabled={!Object.keys(assetsOnNetwork).length}>Settle</ButtonText>
+        </HeaderButton>
+      </ListHeaderWrapper>
+    );
+  };
+
+  navigateToFundTankScreen = async (_: string, wallet: Object) => {
+    const { ensureSmartAccountConnected, navigation } = this.props;
+    this.setState({ showPinScreenForAction: '' });
+
+    await delay(500);
+    ensureSmartAccountConnected(wallet.privateKey)
+      .then(() => {
+        this.setState({ topUpButtonSubmitted: false }, () => navigation.navigate(FUND_TANK));
+      })
+      .catch(() => null);
+  };
+
+  handleCheckPinModalClose = () => {
+    const { resetIncorrectPassword } = this.props;
+    resetIncorrectPassword();
+    this.setState({
+      showPinScreenForAction: '',
+      topUpButtonSubmitted: false,
+    });
+  };
+
+  render() {
+    const { showPinScreenForAction, topUpButtonSubmitted } = this.state;
+    const {
+      availableStake,
+      assetsOnNetwork,
+    } = this.props;
+
+    const assetsOnNetworkArray = Object.keys(assetsOnNetwork).map((asset) => assetsOnNetwork[asset]);
+    const totalStake = availableStake + 10;
+    const availableFormattedAmount = formatMoney(availableStake, 4);
+    const assetsOnNetworkToShow = unionBy(assetsOnNetworkArray, this.initialAssets, 'symbol');
+
+    return (
+      <ScrollView
+        style={{ flex: 1 }}
+        contentContainerStyle={{ flexGrow: 1 }}
+        refreshControl={
+          <RefreshControl
+            refreshing={false}
+            onRefresh={() => {}}
+          />
+        }
+      >
+        <TankBar
+          maxValue={totalStake}
+          currentValue={availableStake}
+          currentValueFormatted={availableFormattedAmount}
+          topupAction={() => this.setState({ showPinScreenForAction: FUND_TANK, topUpButtonSubmitted: true })}
+          topUpLoading={topUpButtonSubmitted}
+        />
+        { /* <AssetButtonsWrapper>
```

Comment:
Not needed or maybe missing any additional comment why it should stay? As well with related commented code in above.

Reply:
It's for Send and Receive buttons. Send flow is not yet implemented - hence I've commented UI for now.
Jegor will work on the flow today.